src/Microsoft.AspNetCore.Authentication.Core/AuthenticationCoreServiceCollectionExtensions.cs

```diff
@@ -24,6 +24,7 @@ public static IServiceCollection AddAuthenticationCore(this IServiceCollection s
                 throw new ArgumentNullException(nameof(services));
             }
 
+            services.AddOptions();
```

Comment:
Hosting always adds this, no? If you're just adding it for tests then add it in the tests.